src/renderer/modals/permission/permissionModal.tsx

```diff
@@ -67,52 +67,74 @@ class PermissionModal extends React.PureComponent<Props, State> {
         };
 
         return (
-            <>
-                <FormattedMessage
-                    id='renderer.modals.permission.permissionModal.body'
-                    defaultMessage={'A site that\'s not included in your Mattermost server configuration requires access for {permission}.'}
-                    values={{
-                        permission: this.props.intl.formatMessage({id: PERMISSION_DESCRIPTION[permission!]}),
-                    }}
-                />
-                <p/>
-                <FormattedMessage
-                    id='renderer.modals.permission.permissionModal.requestOriginatedFromOrigin'
-                    defaultMessage='This request originated from <link>{origin}</link>'
-                    values={{
-                        origin: originDisplay,
-                        link: (msg: React.ReactNode) => (
-                            <a
-
-                                onClick={click}
-                                href='#'
-                            >
-                                {msg}
-                            </a>
-                        ),
-                    }}
-                />
-            </>
+            <div>
+                <p>
+                    <FormattedMessage
+                        id='renderer.modals.permission.permissionModal.body'
+                        defaultMessage={'A site that\'s not included in your Mattermost server configuration requires access for {permission}.'}
+                        values={{
+                            permission: this.props.intl.formatMessage({id: PERMISSION_DESCRIPTION[permission!]}),
+                        }}
+                    />
+                    {}
+                </p>
+                <p>
+                    <FormattedMessage
+                        id='renderer.modals.permission.permissionModal.requestOriginatedFromOrigin'
+                        defaultMessage='This request originated from <link>{origin}</link>'
+                        values={{
+                            origin: originDisplay,
+                            link: (msg: React.ReactNode) => (
+                                <a
+
+                                    onClick={click}
+                                    href='#'
+                                >
+                                    {msg}
+                                </a>
+                            ),
+                        }}
+                    />
+                </p>
+            </div>
         );
     }
 
     render() {
         return (
             <Modal
```

Comment:
Why change this to a Bootstrap modal when the plan is to remove that? Is it because the modal design needs to be updated which would be outside the scope of this PR?

Reply:
This appears to be a merging issue from when I migrated the other modals.